### DISCUSSION TITLE
Spam, Spam Filters and Domain Reputation

### DISCUSSION THREAD

**Discussion by IanNorris** [simple-login/app#879] [Feature Requests]
My first experience with SimpleLogin started by me fishing the confirmation email out of my spam folder, and every email I've had from noreply@ has gone straight to spam, despite me telling Outlook (and Office 365) to whitelist it.

I've been pondering why that is and why the domain rep is so bad (when users can only send emails from different alias domains), when I noticed what the domain of the reply addresses is...

When users report spam in their clients (or clients auto report), they're reporting **simplelogin.co** not the original domain.

If my understanding of this is correct, the only way to fully fix this is to keep the from address the original, and instead use the Reply-To field?

If that's not possible with the architecture, the other way might be to a) tell users to not report emails as spam from their client and instead provide a way to do this on the backend, and b) give each user their own subdomain (possibly from their own domains if they use one?) for reply addresses/reverse aliases that way any issues with the spam filter are contained to a given user.

The other thing to do is move the account and official emails to a different domain that is exclusively used for that purpose - with no user controlled use at all. That way your account emails go to the inbox rather than the spam folder.

Thoughts?

**Comment by nguyenkims** [comment]
> When users report spam in their clients (or clients auto report), they're reporting simplelogin.co not the original domain.

That's true. The reply-to header isn't compatible with multi recipient email unfortunately and that's why we chose the reverse alias approach instead.

> If that's not possible with the architecture, the other way might be to a) tell users to not report emails as spam from their client and instead provide a way to do this on the backend, and b) give each user their own subdomain (possibly from their own domains if they use one?) for reply addresses/reverse aliases that way any issues with the spam filter are contained to a given user.

This is indeed a good idea. However it seems Microsoft uses IP address in their spam algorithm and so using a different subdomains for sending emails wouldn't maybe effective. We chose to enforce a) instead.

> The other thing to do is move the account and official emails to a different domain that is exclusively used for that purpose - with no user controlled use at all. That way your account emails go to the inbox rather than the spam folder.

This is actually already the case where the domain that sends emails is SimpleLogin.co whereas our company domain is rather simplelogin.io

**Comment by xyxzxyz** [comment]
This is my experience as well. Coming from Fastmail, ability to report spam is what I miss the most. Now I receive spams in my inbox that Proton and Fastmail won't filter because they are coming from SimpleLogin but I am not able to report them because that'll report simplelogin instead of original spam. I do hope someday in the future that SL will have Spam filtering function like what other email providers do to fight spammers/scammers and help others.

**Reply by TheFou** [reply]
I don't get it, what's the point of being able to report spam at all ??
Isn't the goal of aliases to just ditch / disable them when they get spam and create a new one ?
And in the process, identify the culprit who breached your email to spammers...

**Reply by IanNorris** [reply]
I've got several email addresses, many of which are on custom domains that I owned prior to switching to the alias approach. Unfortunately one particular email address got leaked in the Kickstarter breach and now I'm getting a ton of Kickstarter spam.

I can't burn that alias (I've been trying for over 3 years to get rid of it), so while I'm still receiving it, I'd love to be able to report it - especially in the case of phishing. Perhaps a forward to spam@simplelogin.co or something?

For the other users (family) on my domains, I keep repeatedly telling them not to report things as spam as their email account is just an alias on SL, but they don't listen.

**Reply by nguyenkims** [reply]
Having an address like spam@simplelogin.co that you can forward spams to is indeed an interesting idea! We need to find a way to use  this collective intelligence in our spam filter though.

**Comment by xyxzxyz** [comment]
Someone suggested whitelisting function here https://github.com/simple-login/app/discussions/726, this could work as alternative.
